tutorials/deploy-remote-desktop-ubuntu-2204/index.mdx

```diff
@@ -0,0 +1,175 @@
+---
+meta:
+  title: Deploy a remote desktop with XRDP on Ubuntu Jammy Jellyfish (22.04 LTS)
+  description: This page shows how to deploy a remote desktop environment with xfce and xrdp on Jammy Jellyfish (22.04 LTS)
+  h1: Deploy a remote desktop with XRDP on Ubuntu Jammy Jellyfish (22.04 LTS)
+  paragraph: This page shows how to deploy a remote desktop environment with xfce and xrdp on Jammy Jellyfish (22.04 LTS)
+categories:
+  - compute
+  - instances
+tags: remote-desktop remote session xRDP rdp desktop pc xFCE jammy-jellyfish
+dates:
+  validation: 2022-05-06
+  posted: 2022-05-06
+---
+
+## Remote Desktop - Overview
+
+A cloud instance can be used with third party software to run a remote desktop. The software captures the Instance's screen and transmits it to another device, where the user can interact with the remote machine using its local keyboard and mouse. 
+
+A remote desktop can be useful for tasks such as:
+* access to a remote computer from anywhere 
+* editing and sharing files between other machines using a high-speed 9nternet connection
+* perform admistrative tasks
+* perform live demonstations of applications installed on the remote machine
+
+In this tutorial you learn deploying a remote desktop on a Instance running on Ubuntu Jammy Jellyfish (22.04 LTS) by using the following stack: 
+* [xRDP](http://xrdp.org/) - an open-source alternative to Microsoft RDP (Microsoft Remote Desktop Protocol).
+* [xfce](https://xfce.org/) - a lightweight and stable desktop environment 
+
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
+- You have an Instance running on Ubuntu Jammy Jellyfish (22.04 LTS)
+- You have sudo privileges or access to the root user
+
+</Message>
+
+## Preparing the remote machine
+
+1. Connect to your Instance using `ssh`:
+
+  ```
+  ssh root@<your_instance_ip>
+  ```
+
+2. Update the `apt` sources list and upgrade the software already installed on the instance:
+
+  ```
+  apt update && apt upgrade -y
+  ```
+
+3. Install the required packages for the desktop environment using the `apt` package manager:
+
+  ```
+  apt install xorg xfce4 xfce4-goodies dbus-x11 x11-xserver-utils
+  ```
+
+4. The xRDP server binary is available in the default Ubuntu repositories and it can be installed using `apt`. The following command installs the packages `xrdp` and `ufw`, a firewall to protect your Instance from unauthorized access:
```

Comment:
```suggestion
The xRDP server binary is available in the default Ubuntu repositories and it can be installed using `apt`.

4. Run the following command installs the packages `xrdp` and `ufw`, a firewall to protect your Instance from unauthorized access:
```